website/docs/configuration/functions/filemd5.html.md

```diff
@@ -0,0 +1,17 @@
+---
+layout: "functions"
+page_title: "filemd5 - Functions - Configuration Language"
+sidebar_current: "docs-funcs-crypto-filemd5"
+description: |-
+  The filemd5 function computes the MD5 hash of the contents of
+  a given file and encodes it as hex.
+---
+
+# `filesha1` Function
```

Comment:
I believe you wanted `filemd5` here

Reply:
Hah... I _knew_ I was going to mess up at least one of these. Thanks!